### ISSUE TITLE
getorf.acd line 0: ACD file not opened

### ISSUE THREAD

**Issue by ShuqingL** [ebi-pf-team/interproscan#122]
I want to annotate my nucleic acid sequence. but run error.

And I read the source core of interproscan and find that before annotation there is a step to translate the nucleic acid sequence to amino acid by bin/nucleotide/getorf :
bin/nucleotide/getorf -sf pearson -sequence /my/input/nucleic.acid.fa -outseq /my/output/amino.acid.fa -minsize 75

but show below error:
Error: File /home/nuka/workpad/getorf/EMBOSS-6.6.0/emboss/acd/getorf.acd line 0: ACD file not opened

actually, "getorf.acd" file is in the same directory of bin/nucleotide/getorf. I don't know why it will use "/home/nuka/workpad/getorf/EMBOSS-6.6.0/emboss/acd/getorf.acd" and this file is not existes.

**Comment by ShuqingL** [comment]
I was do the test refers to https://github.com/ebi-pf-team/interproscan/wiki/ScanNucleicAcidSeqs

**Comment by ShuqingL** [comment]
I do the test refers to https://github.com/ebi-pf-team/interproscan/wiki/ScanNucleicAcidSeqs

**Comment by ShuqingL** [comment]
I don't know why. but I had install EMBOSS in other path before, and when I removed the interproscan getorf binary and replaced it with a link to the stand alone getorf of EMBOSS, it work correctly.

**Comment by ShuqingL** [comment]
I don't know why. but I had install EMBOSS in other path before, and when I removed the interproscan getorf binary and replaced it with a link to the stand alone getorf of EMBOSS, it work correctly.

If you get many warnings like this: 
Warning: Knowntype 'file' in file /path/of/interproscan/bin/nucleotide/knowntypes.standard line 3 - EDAM badly formatted 'EDAM: Amino acid index'
Warning: Knowntype 'file' in file /path/of/interproscan/bin/nucleotide/knowntypes.standard line 4 - EDAM badly formatted 'EDAM: AAindex database'
Warning: Knowntype 'file' in file /path/of/interproscan/bin/nucleotide/knowntypes.standard line 5 - EDAM badly formatted 'EDAM: ABI sequencing trace'
Warning: Knowntype 'file' in file /path/of/interproscan/bin/nucleotide/knowntypes.standard line 7 - EDAM badly formatted 'EDAM: Multiple sequence alignment record'

You may removed the bin/nucleotide/knowntypes.standard file and replaced it with a link to the knowntypes.standard file of EMBOSS you install (like /path/of/EMBOSS/share/EMBOSS/acd/knowntypes.standard).

**Comment by gsn7** [comment]
what version was causing you this problem? the default setup should work well. your problems might have to do with how you have setup to run interproscan.  
If you changed  interproscan.sh file for example then you may not have the correct EMBOSS_ACDROOT path

**Comment by ShuqingL** [comment]
The version is 5.28-67.0.  
I run interproscan in two difference analysis path with all same the input and arguments. But one run correctly and another not. I don't know why too. But this problem may be other topic. And now I try to install the latest version.

**Comment by ShuqingL** [comment]
I install the new interproscan, but get the same problem. 
It is because I have a getorf in /usr/bin? (/usr/bin/getorf)

**Comment by gsn7** [comment]
as this has to do with your environment, so go with what works for you. 
maybe change the value of the following property in the interproscan.properties file to /usr/bin/getorf
binary.getorf.path=

**Comment by ShuqingL** [comment]
When modify binary.getorf.path value to /usr/bin/getorf, can not work too.
below is the error output.
(It is work if do protein annotation "./interproscan.sh -i test_proteins.fasta -f tsv" )

$ ./interproscan.sh -i test_nt_seqs.fasta -f tsv -t n
04/12/2019 20:21:57:290 Welcome to InterProScan-5.39-77.0
04/12/2019 20:21:57:292 Running InterProScan v5 in STANDALONE mode... on Linux
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - Execution thrown when attempting to executeInTransaction the StepExecution.  All database activity rolled back.
java.lang.NullPointerException
        at uk.ac.ebi.interpro.scan.management.model.implementations.RunBinaryStep.execute(RunBinaryStep.java:141)
        at uk.ac.ebi.interpro.scan.jms.activemq.StepExecutionTransactionImpl.executeInTransaction(StepExecutionTransactionImpl.java:86)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:198)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
        at com.sun.proxy.$Proxy142.executeInTransaction(Unknown Source)
        at uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener.onMessage(LocalJobQueueListener.java:193)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:761)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:699)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:674)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.doReceiveAndExecute(AbstractPollingMessageListenerContainer.java:318)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.receiveAndExecute(AbstractPollingMessageListenerContainer.java:257)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.invokeListener(DefaultMessageListenerContainer.java:1189)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.executeOngoingLoop(DefaultMessageListenerContainer.java:1179)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1076)
        at java.base/java.lang.Thread.run(Thread.java:834)
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - The exception is :
java.lang.NullPointerException
        at uk.ac.ebi.interpro.scan.management.model.implementations.RunBinaryStep.execute(RunBinaryStep.java:141)
        at uk.ac.ebi.interpro.scan.jms.activemq.StepExecutionTransactionImpl.executeInTransaction(StepExecutionTransactionImpl.java:86)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:198)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
        at com.sun.proxy.$Proxy142.executeInTransaction(Unknown Source)
        at uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener.onMessage(LocalJobQueueListener.java:193)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:761)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:699)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:674)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.doReceiveAndExecute(AbstractPollingMessageListenerContainer.java:318)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.receiveAndExecute(AbstractPollingMessageListenerContainer.java:257)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.invokeListener(DefaultMessageListenerContainer.java:1189)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.executeOngoingLoop(DefaultMessageListenerContainer.java:1179)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1076)
        at java.base/java.lang.Thread.run(Thread.java:834)
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - 2. The exception is :java.lang.NullPointerException
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - StepExecution with errors - stepName: stepRunGetOrf
04/12/2019 20:22:15:576 Loading file /share/Software/interproscan-5.39-77.0/test_nt_seqs.fasta
04/12/2019 20:22:15:580 Running the following analyses:
[jobTIGRFAM,jobSFLD,jobSuperFamily-1.75,jobPanther,jobGene3d,jobHAMAP,jobCoils,jobPrositeProfiles,jobSMART,jobCDD,jobPRINTS-42.0,jobPrositePatterns,jobPfam,jobMobiDBLite,jobPIRSF]
04/12/2019 20:22:15:580 Running the following analyses:
[CDD-3.17,Coils-2.2.1,Gene3D-4.2.0,Hamap-2019_01,MobiDBLite-2.0,PANTHER-14.1,Pfam-32.0,PIRSF-3.02,PRINTS-42.0,ProSitePatterns-2019_01,ProSiteProfiles-2019_01,SFLD-4,SMART-7.1,SUPERFAMILY-1.75,TIGRFAM-15.0]
FATAL: uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError - Analysis step 5 : Run GetOrf to predict open reading frames from nucleic acid sequence for proteins null to null has failed irretrievably.  Available StackTraces follow.
FATAL: uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError - The JVM will now exit with a non-zero exit status.
ERROR: uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster - Exception thrown by StandaloneBlackBoxMaster:
java.lang.IllegalStateException: InterProScan exiting with non-zero status, see logs for further information.
        at uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError.failed(NonZeroExitOnUnrecoverableError.java:42)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.run(StandaloneBlackBoxMaster.java:147)
        at uk.ac.ebi.interpro.scan.jms.main.Run.main(Run.java:460)
java.lang.IllegalStateException: InterProScan exiting with non-zero status, see logs for further information.
        at uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError.failed(NonZeroExitOnUnrecoverableError.java:42)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.run(StandaloneBlackBoxMaster.java:147)
        at uk.ac.ebi.interpro.scan.jms.main.Run.main(Run.java:460)
WARN : uk.ac.ebi.interpro.scan.jms.master.AbstractMaster - Unable to delete temporary directory /share/Software/interproscan-5.39-77.0/temp/health_login_20191204_202203711_2iew
java.io.IOException: Unable to delete directory /share/Software/interproscan-5.39-77.0/temp/health_login_20191204_202203711_2iew/kvstore/matchDB.
        at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1623)
        at org.apache.commons.io.FileUtils.forceDelete(FileUtils.java:2391)
        at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:1721)
        at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1617)
        at org.apache.commons.io.FileUtils.forceDelete(FileUtils.java:2391)
        at org.apache.commons.io.FileUtils.cleanDirectory(FileUtils.java:1721)
        at org.apache.commons.io.FileUtils.deleteDirectory(FileUtils.java:1617)
        at org.apache.commons.io.FileUtils.forceDelete(FileUtils.java:2391)
        at uk.ac.ebi.interpro.scan.jms.master.AbstractMaster.deleteWorkingTemporaryDirectory(AbstractMaster.java:249)
        at uk.ac.ebi.interpro.scan.jms.master.AbstractMaster.cleanUpWorkingDirectory(AbstractMaster.java:264)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.systemExit(StandaloneBlackBoxMaster.java:340)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.run(StandaloneBlackBoxMaster.java:302)
        at uk.ac.ebi.interpro.scan.jms.main.Run.main(Run.java:460)
InterProScan analysis failed. Exception thrown by StandaloneBlackBoxMaster. Check the log file for details

**Comment by gsn7** [comment]
maybe stick what has worked for you as you described above - remove the interproscan getorf binary and replace it with a link to the stand alone getorf of EMBOSS.
there is something happening with your environment but don't know what it is

**Comment by ShuqingL** [comment]
I have replaced getorf to a absolute link getorf of emboss. If not, I will get another error information.
Is it because that I don't "Install/configure the Pre-calculated Match Lookup Service" when I install InterProScan? Isn't that an option?

**Comment by ShuqingL** [comment]
I try to translate amino acid by myself, and the output sequence id is like below:
CL96.Contig6_All.p1 type:complete len:935 gc:universal CL96.Contig6_All:406-3210(+)
CL96.Contig6_All.p2 type:complete len:79 gc:universal CL96.Contig6_All:1562-1326(-)
CL96.Contig6_All.p3 type:complete len:74 gc:universal CL96.Contig6_All:623-844(+)
CL96.Contig6_All.p4 type:complete len:74 gc:universal CL96.Contig6_All:1721-1942(+)

and if I run "sh interproscan.sh -i input.pep -cpu 20 -f TSV -t p -goterms --outfile output.tsv", there are still error.
But after I change the sequence id to "Gene1", "Gene2" ... interproscan.sh run correctly.  If someone has the same question, I hope this answer will help. Although I still don't know why.

**Comment by davidbio** [comment]
Hi there,
I get a similar error message regarding the getorf executable, please see below.
It only happens with the latest InterProScan package.

I also noticed that the EMBOSS getorf binary, I get from compiling it myself, is different from the one shipped with IPS. The binary size and program output are not the same.

Is this supposed to be the same executable?
Can you help with this?

```bash
$ ./interproscan.sh -i test_nt_seqs.fasta -goterms -dp -cpu 4 -f xml -d ~/docker_ips/ips/test -T /tmp -t n                                  
10/12/2019 11:01:28:730 Welcome to InterProScan-5.39-77.0
10/12/2019 11:01:28:732 Running InterProScan v5 in STANDALONE mode... on Linux
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - Execution thrown when attempting to executeInTransaction the StepExecution.  All database activity rolled ba
ck.
java.lang.NullPointerException
        at uk.ac.ebi.interpro.scan.management.model.implementations.RunBinaryStep.execute(RunBinaryStep.java:141)
        at uk.ac.ebi.interpro.scan.jms.activemq.StepExecutionTransactionImpl.executeInTransaction(StepExecutionTransactionImpl.java:86)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:198)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
        at com.sun.proxy.$Proxy142.executeInTransaction(Unknown Source)
        at uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener.onMessage(LocalJobQueueListener.java:193)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:761)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:699)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:674)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.doReceiveAndExecute(AbstractPollingMessageListenerContainer.java:318)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.receiveAndExecute(AbstractPollingMessageListenerContainer.java:257)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.invokeListener(DefaultMessageListenerContainer.java:1189)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.executeOngoingLoop(DefaultMessageListenerContainer.java:1179)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1076)
        at java.base/java.lang.Thread.run(Thread.java:834)
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - The exception is :
java.lang.NullPointerException
at uk.ac.ebi.interpro.scan.management.model.implementations.RunBinaryStep.execute(RunBinaryStep.java:141)
        at uk.ac.ebi.interpro.scan.jms.activemq.StepExecutionTransactionImpl.executeInTransaction(StepExecutionTransactionImpl.java:86)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
        at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
        at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
        at java.base/java.lang.reflect.Method.invoke(Method.java:566)
        at org.springframework.aop.support.AopUtils.invokeJoinpointUsingReflection(AopUtils.java:343)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.invokeJoinpoint(ReflectiveMethodInvocation.java:198)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:163)
        at org.springframework.transaction.interceptor.TransactionAspectSupport.invokeWithinTransaction(TransactionAspectSupport.java:294)
        at org.springframework.transaction.interceptor.TransactionInterceptor.invoke(TransactionInterceptor.java:98)
        at org.springframework.aop.framework.ReflectiveMethodInvocation.proceed(ReflectiveMethodInvocation.java:186)
        at org.springframework.aop.framework.JdkDynamicAopProxy.invoke(JdkDynamicAopProxy.java:212)
        at com.sun.proxy.$Proxy142.executeInTransaction(Unknown Source)
        at uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener.onMessage(LocalJobQueueListener.java:193)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doInvokeListener(AbstractMessageListenerContainer.java:761)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.invokeListener(AbstractMessageListenerContainer.java:699)
        at org.springframework.jms.listener.AbstractMessageListenerContainer.doExecuteListener(AbstractMessageListenerContainer.java:674)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.doReceiveAndExecute(AbstractPollingMessageListenerContainer.java:318)
        at org.springframework.jms.listener.AbstractPollingMessageListenerContainer.receiveAndExecute(AbstractPollingMessageListenerContainer.java:257)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.invokeListener(DefaultMessageListenerContainer.java:1189)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.executeOngoingLoop(DefaultMessageListenerContainer.java:1179)
        at org.springframework.jms.listener.DefaultMessageListenerContainer$AsyncMessageListenerInvoker.run(DefaultMessageListenerContainer.java:1076)
        at java.base/java.lang.Thread.run(Thread.java:834)
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - 2. The exception is :java.lang.NullPointerException
ERROR: uk.ac.ebi.interpro.scan.jms.worker.LocalJobQueueListener - StepExecution with errors - stepName: stepRunGetOrf
10/12/2019 11:01:52:787 Loading file /home/ips/docker_ips/ips/test_nt_seqs.fasta
10/12/2019 11:01:52:789 Running the following analyses:
[jobTIGRFAM,jobSFLD,jobSuperFamily-1.75,jobPanther,jobGene3d,jobHAMAP,jobCoils,jobPrositeProfiles,jobSMART,jobCDD,jobPRINTS-42.0,jobPrositePatterns,jobPfam,jobMobiDBLite,jobP
IRSF]
10/12/2019 11:01:52:790 Running the following analyses:
[CDD-3.17,Coils-2.2.1,Gene3D-4.2.0,Hamap-2019_01,MobiDBLite-2.0,PANTHER-14.1,Pfam-32.0,PIRSF-3.02,PRINTS-42.0,ProSitePatterns-2019_01,ProSiteProfiles-2019_01,SFLD-4,SMART-7.1
,SUPERFAMILY-1.75,TIGRFAM-15.0]
ERROR: uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster - Exception thrown by StandaloneBlackBoxMaster: 
java.lang.IllegalStateException: InterProScan exiting with non-zero status, see logs for further information.
        at uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError.failed(NonZeroExitOnUnrecoverableError.java:42)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.run(StandaloneBlackBoxMaster.java:147)
        at uk.ac.ebi.interpro.scan.jms.main.Run.main(Run.java:460)
java.lang.IllegalStateException: InterProScan exiting with non-zero status, see logs for further information.
        at uk.ac.ebi.interpro.scan.jms.activemq.NonZeroExitOnUnrecoverableError.failed(NonZeroExitOnUnrecoverableError.java:42)
        at uk.ac.ebi.interpro.scan.jms.master.StandaloneBlackBoxMaster.run(StandaloneBlackBoxMaster.java:147)
        at uk.ac.ebi.interpro.scan.jms.main.Run.main(Run.java:460)
InterProScan analysis failed. Exception thrown by StandaloneBlackBoxMaster. Check the log file for details
```

**Comment by gsn7** [comment]
@davidbio there is a bug in version 5.39-77.0 in dealing with nucleotide sequences. we will make a fix

**Comment by gsn7** [comment]
this bug is fixed in release 5.41-78.0
